docs/src/main/paradox/file.md

```diff
@@ -141,20 +119,20 @@ Scala
 :   ```
     sbt
     // tail source
-    > akka-stream-alpakka-file/test:runMain akka.stream.alpakka.file.scaladsl.FileTailSourceSpec /some/path/toa/file
+    > file/Test/runMain akka.stream.alpakka.file.scaladsl.FileTailSourceSpec /some/path/toa/file
```

Comment:
Good find, do people find the "Running the code" useful?

Reply:
The fact that no one discovered this before seems to indicate that not so much.